melior_transformers/classification/classification_utils.py

```diff
@@ -452,9 +452,9 @@ def delete_worst_models(args, results_path):
     for result in results_dict.keys():
         metric_val = results_dict[result][args["metric_criteria"]]
         results_list.append([result, metric_val])
-    print(results_list)
     # Sort results
     results_list.sort(key=lambda x: x[1], reverse=True)
+    print(results_list)
```

Comment:
It is just for debug purpose in case it fails. Hope we can change it to logger soon.